install-manifests/docker-compose-traefix-tls-gzip/docker-compose.yml

```diff
@@ -0,0 +1,100 @@
+version: '3.6'
+
+services:
+  traefix:
+    image: traefik # The official Traefik docker image
+    network_mode: bridge
+    command: >
+      # Enables the web UI and tells Traefik to listen to docker
+      --api \
+      --docker --docker.watch --docker.domain=docker.localhost \
+      --ping --ping.entryPoint=http \
+      --metrics.prometheus \
+      --entrypoints='Name:http Address::80 Redirect.EntryPoint:https' \
+      --entryPoints='Name:https Address::443 TLS' \
+      --acme \
+      --acme.acmelogging \
+      --acme.caserver=${LETS_ENCRYPT_ACME_SERVER} --acme.email=${LETS_ENCRYPT_EMAIL}   \
+      --acme.storage=/acme.json \
+      --acme.entrypoint=https \
+      --acme.onhostrule  --acme.httpchallenge \
+      --acme.httpchallenge.entrypoint=http \
+      --retry \
+      --loglevel=DEBUG
+    restart: unless-stopped
+    networks:
+      - web
+    environment:
+      LETS_ENCRYPT_ACME_SERVER: 'https://acme-staging-v02.api.letsencrypt.org/directory'
+      LETS_ENCRYPT_EMAIL: your@email.com
+    ports:
+      - '80:80' # The HTTP port
+      - '443:443' # HTTPS Port (TLS)
+      - '8080:8080' # The Web UI (enabled by --api)
```

Comment:
Is the web UI protected?

Reply:
I have added BASIC Auth for the web UI. Thanks for feedback.